### PR TITLE
Dropping Corpses Devoured by Space Dragons on Gib/Butcher. 

### PR DESCRIPTION
**Pull request by Lyndomen**
## About the PR
Space Dragon upon being gibbed/butchered now drops the bodies of those they have consumed. This PR is credited to user Mnemotechnician, who originally submitted it downstream to DV. https://github.com/DeltaV-Station/Delta-v/pull/1308

They have given their permission to PR this upstream under your MIT license.

## Why / Balance
Bug fix to issue https://github.com/space-wizards/space-station-14/issues/20405

## Technical details
Made DevourSystem empty the stomach container of the Devourer when it's being gibbed (butchering includes gibbing).

## Media

https://github.com/space-wizards/space-station-14/assets/49795619/a5f44c84-aa2b-43b7-90db-ae3b3f84d911

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- fix: Space Dragons will drop bodies upon being gibbed/butchered

### CODE REVIEW

**Comment by Brandon-Huu** [comment]
change log kinda weird

**Comment by Kadeo64** [comment]
only took 8 months

**Review by ShadowCommander — COMMENTED** [review]
Looks good. Could you specify MIT as the license in the description?

File: Content.Server/Devour/DevourSystem.cs [hunk 1]
```diff
@@ -45,5 +47,15 @@ private void OnDoAfter(EntityUid uid, DevourerComponent component, DevourDoAfter
 
         _audioSystem.PlayPvs(component.SoundDevour, uid);
     }
+    
+        private void OnGibContents(EntityUid uid, DevourerComponent component, ref BeingGibbedEvent args)
+    {
```
Comment (metalgearsloth) [inline Content.Server/Devour/DevourSystem.cs:52]:
```suggestion
    private void OnGibContents(EntityUid uid, DevourerComponent component, ref BeingGibbedEvent args)
    {
```

**Pull request merged**